x-pack/plugins/security/server/lib/authorization/privileges.js

```diff
@@ -43,7 +44,8 @@ export function buildPrivilegeMap(savedObjectTypes, actions) {
           'get',
           'bulk_get',
           'find',
-          'update'
+          'update',
+          'find_relationships'
```

Comment:
Lets use trailing commas for the updates in this file, and can we keep `find_relationships` next to `find` here?